src/actions/entries.js

```diff
@@ -288,7 +288,7 @@ export function persistEntry(collection) {
           kind: 'success',
           dismissAfter: 4000,
         }));
-        return dispatch(entryPersisted(collection, serializedEntry));
+        return Promise.resolve(dispatch(entryPersisted(collection, serializedEntry)));
```

Comment:
I don't think the `Promise.resolve` here isn't doing anything - `dispatch(entryDeleted(...))` should already be returning a promise.

Reply:
Good point -- since its returning from a `then`, it already gets wrapped in a promise.